zkevm-circuits/src/evm_circuit/param.rs

```diff
@@ -1,9 +1,21 @@
+use super::table::Table;
+
 // Step dimension
-pub(crate) const STEP_WIDTH: usize = 32;
+pub(crate) const STEP_WIDTH: usize = 128;
```

Comment:
Pretty much yeah. Because most opcodes currently still take up at least 2 rows I would say it's very likely that making the circuit even wider (more columns and/or extra lookups per row) so most cheap (cheap as in gas cost) opcodes take up 1 row will very likely be better.